src/monitor/node_active_protocol.c

```diff
@@ -261,14 +262,21 @@ node_active(PG_FUNCTION_ARGS)
 
 	checkPgAutoFailoverVersion();
 
+	ereport(INFO, (errmsg("node_active is called with %s, %d, " UINT64_FORMAT ,
+						   nodeName, nodePort,  currentLSN)));
+
+
 	currentNodeState.nodeId = currentNodeId;
 	currentNodeState.groupId = currentGroupId;
 	currentNodeState.replicationState =
 		EnumGetReplicationState(currentReplicationStateOid);
-	currentNodeState.walDelta = currentWalDelta;
+	currentNodeState.reportedLSN = currentLSN;
 	currentNodeState.pgsrSyncState = SyncStateFromString(currentPgsrSyncState);
 	currentNodeState.pgIsRunning = currentPgIsRunning;
 
+	ereport(INFO, (errmsg("before calling internal NodeActive with  %s, %d, " INT64_FORMAT ,
+						   nodeName, nodePort, currentNodeState.reportedLSN )));
+
```

Comment:
Looks like debug logs, do we need them once the PR is merged?

Reply:
removed